src/ContainerExceptionInterface.php

```diff
@@ -8,6 +8,6 @@
 /**
  * Base interface representing a generic exception in a container.
  */
-interface ContainerExceptionInterface
+interface ContainerExceptionInterface extends \Throwable
```

Comment:
This change breaks a lot of [symfony installations](https://github.com/symfony/symfony/search?q=ContainerExceptionInterface+Throwable).

Reply:
Seems fixed since PHP 7.4.0+

https://3v4l.org/vUsLI

Reply:
You're correct, but 7.3 it's not yet EOL. 

And thank you for the link, I wasn't aware of that change in 7.4.

As a BC change I wonder why it wasn't shipped with 2.0.0.

Reply:
Honestly, we missed this; it's a language bug that we weren't expecting. We will probably reintroduce the change bumping the PHP requirement to 7.4, see #32